docs/migrating-from-legacy-packages.md

```diff
@@ -0,0 +1,606 @@
+	happo-e2e \
+	happo-plugin-storybook \
+	happo-static
+npm install --save-dev happo
+```
+
+The new `happo` package replaces all of these legacy packages. All functionality
+is now included in the single unified package.
+
+### Configuration File Format
+
+The new package supports multiple configuration file formats and encourages
+TypeScript/ES modules:
+
+**Before (legacy):**
+
+```js title=.happo.js
+const { RemoteBrowserTarget } = require('happo.io');
+
+module.exports = {
+  apiKey: process.env.HAPPO_API_KEY,
+  apiSecret: process.env.HAPPO_API_SECRET,
+  targets: {
+    'chrome-desktop': new RemoteBrowserTarget('chrome', {
+      viewport: '1024x768',
+    }),
+  },
+};
+```
+
+**After (new):**
+
+```js title=happo.config.ts
+import { defineConfig } from 'happo';
+
+export default defineConfig({
+  apiKey: process.env.HAPPO_API_KEY!,
+  apiSecret: process.env.HAPPO_API_SECRET!,
+
+	project: 'default',
+
+	integration: {
+		type: 'storybook',
+		configDir: '.storybook',
+	},
+
+  targets: {
+    'chrome-desktop': {
+      browserType: 'chrome',
+      viewport: '1024x768',
+    },
+  },
+});
+```
+
+### Target Configuration
+
+One of the most significant changes is how targets are defined. The new package
+uses plain objects instead of the `RemoteBrowserTarget` class.
+
+**Before (legacy):**
+
+```js title=.happo.js
+const { RemoteBrowserTarget } = require('happo.io');
+
+module.exports = {
+  // ... rest of config
+
+  targets: {
+    'firefox-desktop': new RemoteBrowserTarget('firefox', {
+      viewport: '1024x768',
+      chunks: 2,
+      freezeAnimations: 'last-frame',
+    }),
+    'chrome-mobile': new RemoteBrowserTarget('chrome', {
+      viewport: '375x667',
+    }),
+    'ios-safari': new RemoteBrowserTarget('ios-safari', {
+      viewport: '375x667',
+    }),
+  },
+};
+```
+
+**After (new):**
+
+```js title=happo.config.ts
+import { defineConfig } from 'happo';
+
+export default defineConfig({
+  // ... rest of config
+
+  targets: {
+    'firefox-desktop': {
+      browserType: 'firefox',
+      viewport: '1024x768',
+      chunks: 2,
+      freezeAnimations: 'last-frame',
+    },
+    'chrome-mobile': {
+      browserType: 'chrome',
+      viewport: '375x667',
+    },
+    'ios-safari': {
+      browserType: 'ios-safari',
+    },
+  },
+});
+```
+
+**Key changes:**
+
+- Remove `new RemoteBrowserTarget()` wrapper
+- Use `browserType` instead of the first constructor argument
+- All target options remain the same (viewport, chunks, freezeAnimations, etc.)
+- Mobile Safari targets (`ios-safari`, `ipad-safari`) don't require a viewport
+  (they use a fixed size)
+- `freezeAnimations` now defaults to `'last-frame'`
+
+### CLI Commands
+
+The CLI command changes as well:
+
+**Before (legacy):**
+
+```bash
+npx happo run
+# or
+npm run happo run
+```
+
+**After (new):**
+
+```bash
+npx happo
+# or
+npm run happo
+```
+
+The CLI is now provided by the `happo` package instead of `happo.io` and has a
+simplified API.
```

Comment:
We should list the `happo-ci-*` commands here as well. `happo-ci-github-actions`, `happo-ci-circleci`, `happo-ci-travis`, `happo-ci-azure-pipelines`.